src/vs/editor/contrib/suggest/browser/suggestModel.ts

```diff
@@ -409,7 +415,7 @@ export class SuggestModel implements IDisposable {
 		if (ctx.column > this.context.column && this.completionModel.incomplete && ctx.leadingWord.word.length !== 0) {
 			// typed -> moved cursor RIGHT & incomple model & still on a word -> retrigger
 			const { complete, incomplete } = this.completionModel.resolveIncompleteInfo();
-			this.trigger(this._state === State.Auto, true, incomplete, complete);
+			this.trigger({ auto: this._state === State.Auto }, true, incomplete, complete);
```

Comment:
Not so sure about having `auto` in the context... Today this too course grained and we need to differentiate between 24x7 IntelliSense and resolving an incomplete suggestion list. Today, `auto` is more or less just used to control the widget, e.g. show empty with loading, show empty results etc...

At least, we must make sure that this doesn't leak into the extension host